src/V3SplitVar.cpp

```diff
@@ -336,30 +358,11 @@ class UnpackRefMap {
     MapIt end() { return m_map.end(); }
 };
 
-// Compare AstNode* to get deterministic ordering when showing messages.
-struct AstNodeComparator {
-    bool operator()(const AstNode* ap, const AstNode* bp) const {
-        const FileLine* afp = ap->fileline();
-        const FileLine* bfp = bp->fileline();
-        if (afp->firstLineno() != bfp->firstLineno())
-            return afp->firstLineno() < bfp->firstLineno();
-        if (afp->firstColumn() != bfp->firstColumn())
-            return afp->firstColumn() < bfp->firstColumn();
-        // Don't comapre its filename because it is expensive.
-        // Line number and column is practically enough.
-        // The comparison of raw pointer here is just in case.
-        // The comparison of this pointer may differ on different run,
-        // but this is just warning message order. (mostly for CI)
-        // Verilated result is same anyway.
-        return ap < bp;
-    }
-};
-
 // Found nodes for SplitPackedVarVisitor
 struct RefsInModule {
-    std::set<AstVar*> m_vars;
-    std::set<AstVarRef*> m_refs;
-    std::set<AstSel*> m_sels;
+    std::set<AstVar*, AstNodeComparator> m_vars;
+    std::set<AstVarRef*, AstNodeComparator> m_refs;
+    std::set<AstSel*, AstNodeComparator> m_sels;
```

Comment:
Pleas make typedefs for these three, then use them here and below so the iterators are cleaner. Thanks.

Reply:
I added typefs in 00ff30b5ad712aa74e6bff5a3c67154195deb820.